project.clj

```diff
@@ -62,7 +62,7 @@
                                   [puppetlabs/trapperkeeper ~tk-version :classifier "test"]
                                   [org.clojure/tools.namespace "0.2.4"]
                                   [org.clojure/java.jmx "0.2.0"]
-                                  [spyscope "0.1.4"]
+                                  [spyscope "0.1.5" :exclusions [clj-time]]
```

Comment:
Ok, but if I do, I'll have to add an exclusion for it on clj-time to it since spyscope 0.1.5 depends upon a newer clj-time.  'Spose that's okay - not sure though if there's some functionality in spyscope that depends upon functionality in a newer clj-time?

dependencies--